apps/sms/services/js/moz_mobile_connections/moz_mobile_connections_shim.js

```diff
@@ -0,0 +1,104 @@
+/* global BridgeServiceMixin, BroadcastChannel, MozSettingsShim */
```

Comment:
__optional nit:__ IIRC we agreed to put every globals on its own line to easier track when new added or old removed.